### ISSUE TITLE
pip3 install get's stuck

### ISSUE THREAD

**Issue by rpanchal1996** [plasticityai/magnitude#14]
Hey, 

This project looks really exciting and kudos on the great work! I am trying to install the `pymagnitude` package, using the command `pip3 install pymagnitude`, but it get's stuck after downloading and doesn't proceed to install. I supplied the verbose flag with the command to see what it's getting stuck at and turn's out, it's getting stuck at:
```
Running setup.py (path:/tmp/pip-build-60o8i7g_/pymagnitude/setup.py) egg_info for package pymagnitude
Running command python setup.py egg_info
```
I tried installing it in a fresh virtual environment just to ensure that it was not the fault of my environment, but it did not work there too. Is it actually getting stuck or am I just being impatient and the package take 5+ minutes to install?

For your reference, I am running Python 3.6.5 on an Ubuntu 16.04

Any help would be appreciated. Thank you!

**Comment by AjayP13** [comment]
Just released version 0.1.46. You can now disable downloading the wheel and force it to compile instead by running this command in your shell instead:

```bash
SKIP_MAGNITUDE_WHEEL=1 pip3 install pymagnitude==0.1.46 -vvvv
````

I think the downloading and installing of the wheel is just taking extra long on your computer / network, but if you really need to skip it, try running this command.

Note: It is actually slower than installing a wheel since it will have to run GCC to compile some large C code instead of using a pre-built binary. But it seems like the bottleneck on your computer is downloading the wheel.

**Comment by rpanchal1996** [comment]
This did the job for me. Thank you so much for your prompt response!
